docs/rules/prefer-object-has-own.md

```diff
@@ -0,0 +1,53 @@
+# Prefer `Object.hasOwn()` over `Object.prototype.hasOwnProperty.call()` (prefer-object-has-own)
+
+It is very common to write code like:
+
+```js
+if (Object.prototype.hasOwnProperty.call(object, "foo")) {
+  console.log("has property foo");
+}
+```
+
+This is a common practice because methods on Object.prototype can sometimes be unavailable or redefined (see the [no-prototype-bultins](no-prototype-builtins.md) rule).
```

Comment:
```suggestion
This is a common practice because methods on `Object.prototype` can sometimes be unavailable or redefined (see the [no-prototype-builtins](no-prototype-builtins.md) rule).
```